e2e/logger-e2e/tests/logger.spec.ts

```diff
@@ -47,5 +50,15 @@ describe('logger e2e', () => {
                 ).not.toThrow();
             });
         });
+
+        describe('test logger', () => {
+            it('should run logger tests', async () => {
+                const project = uniq('logger');
+                await runNxCommandAsync(
+                    `generate @ensono-stacks/logger:winston ${project}`,
+                );
+                expect(await runTarget(project, targetOptions.test)).toContain('Successfully ran target test');
```

Comment:
Does the logger have any other targets, linting?

Reply:
It doesn't look like logger has any other commands to run, you can only do build and test.

As for other output for test failures. We should only get that text if all tests pass from what I can see. What could maybe check that it doesn't contain something, maybe something like Fail/Failed.

Reply:
In which case, could we also get a test added here for building?

Reply:
Added additional test for build logger